api/build.gradle

```diff
@@ -60,6 +60,7 @@ dependencies {
     testCompile "org.junit.vintage:junit-vintage-engine:${junit5Version}"
     testCompile "org.testcontainers:postgresql:${testcontainersVersion}"
     testCompile "org.testcontainers:junit-jupiter:${testcontainersVersion}"
+    testImplementation 'org.apache.httpcomponents:httpclient:4.5.13'
```

Comment:
We'll want to use `openlineage-java` and replace the internal implementation. I've opened an issue as follow up work once this PR has been merge, see https://github.com/MarquezProject/marquez/issues/1579